client/bin/smee.js

```diff
@@ -7,11 +7,17 @@ const Client = require('..')
 program
   .usage('[options]')
   .option('-u, --url <url>', 'URL of the webhook proxy service. Default: https://smee.io/new')
+  .option('-t, --target <target>', 'URL of the target service the events will forwarded to. Default: http://127.0.0.1:port/path')
   .option('-p, --port <n>', 'Local HTTP server port', process.env.PORT || 3000)
   .option('-P, --path <path>', 'URL path to post proxied requests to`', '/')
   .parse(process.argv)
 
-const target = `http://127.0.0.1:${program.port}${program.path}`
+let target
+if (program.target) {
+  target = program.target
```

Comment:
Good idea @pepibumur! Is the intention of this option to completely replace using `port` and `path`? Currently the way you're constructing it, `program.target` would have to be the full URL, including the `http(s)` and any port/path values. I assume because in a docker environment you'd be forwarding to a service exposed on something like `https://${DNS_NAME}`?